mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -994,6 +1170,7 @@
         <SubgroupPriority value="36"/>
         <GlossaryStrongArray index="1" value="LurkerMP"/>
         <FlagArray index="UseLineOfSight" value="1"/>
+        <Speed value="2.0156"/>
```

Comment:
Observer's move speed increased from 2.63 to 2.82 (from 1.875 to 2.0156 on normal speed)

move speed is tiles per second. The patch notes translated it to real seconds on the multiplayer game speed (x1.4)